### ISSUE TITLE
How to add a dynamic route to the access map?

### ISSUE THREAD

**Issue by xShirase** [krakenjs/kraken-example-with-passport#7]
Hi, and thanks for this extremely educational tutorial!

One question, I want to protect a route that looks like "/list/:listname", how do I add that to the access maps in auth.js? Do I have to? 

Thanks again, Kraken is damn awesome!

**Comment by pvenkatakrishnan** [comment]
hey @xShirase, just to confirm i understand the question,  you want to selectively add auth middleware to routes which have `/list/:listname` and you have other routes you dont want to, yeah ?

**Comment by pvenkatakrishnan** [comment]
And thanks for `Kraken is damn awesome!`

**Comment by xShirase** [comment]
Yes, the route I'm trying to protect is defined as: 
`router.get('/list/:listname' ...)`

**Comment by pvenkatakrishnan** [comment]
Also checkout this sample app. https://github.com/jasisk/middleware-patterns
especially https://github.com/jasisk/middleware-patterns/blob/master/config/whitelist.json ,seems like what you are trying to do.

**Comment by xShirase** [comment]
Seems like it yes, I'll investigate that.
I've chosen Kraken for my startup launching soon, and it's a pleasure to work with.
However, the main drawback I find is the lack of documentation about all the conventions you've added on top of Express. A wannabe Kraken user has a lot of googling to do to figure out that it's really simple :)
I'd be willing to help writing some docs/tutorials to help others like me as I go further with the framework, is there a centralized wiki for that or should I sort it out myself?

**Comment by pvenkatakrishnan** [comment]
Thank you for the feedback @xShirase We agree and we are working on that actively now.  Any help is deeply appreciated.
Have you checked out krakenjs.com ? We are trying to revamp the site and make documentation clearer.
If you have specific feedback on areas you'd like us to improve, please help us by filing issues against the repos. We really want to help connect the dots and make it easily accessible.

**Comment by xShirase** [comment]
Sounds good. I'll also make sure to document my journey on a dev blog and release that as soon as we've launched. I'll probably run the articles by your team to make sure I don't spread misconceptions.
Kraken is by far the best toolkit out there, leveraging Express makes it way more powerful than Hapi and the others. At the moment, with very little migration effort, our load time has dropped by 30% and we gained all the security and specialization features, I couldn't be happier!
